### PR TITLE
Code cleaning

### PR DESCRIPTION
**Pull request by kuba-end**
Mailchimp plugin fixed with ecs and phpstan

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -9,3 +9,5 @@
 !/etc/build/.gitkeep
 
 /tests/Application/yarn.lock
+/.env
+/.env.*
```
Comment (er1z) [inline .gitignore]: IMO too wide, it'll exclude tests configuration.

File: src/Controller/NewsletterController.php [hunk 2]
```diff
@@ -50,7 +50,7 @@ public function subscribeAction(Request $request): JsonResponse
         $email = $request->request->get('email');
         $token = $request->request->get('_token');
 
-        if(!is_string($email) || !is_string($token)){
+        if (!is_string($email) || !is_string($token)) {
```
Comment (er1z) [inline src/Controller/NewsletterController.php:53]: Why this validation isn't done as a form? It'll simplify coding and make it more modular in the future and — also — will remove token validator logic.

File: tests/Application/.gitignore [hunk 3]
```diff
@@ -12,6 +12,7 @@
 /.env.*.local
 /.env.local
 /.env.local.php
+/.env.test
```
Comment (er1z) [inline tests/Application/.gitignore]: Why?

File: src/Controller/NewsletterController.php (hunk 2 above)
Comment (kuba-end) [reply src/Controller/NewsletterController.php:53]: Is this 'if' for $email necessary if two lines below we using ```$this->validator->validate($email)``` which check if $email is EmailType

**Review by er1z — APPROVED** [review]

File: composer.json [hunk 4]
```diff
@@ -5,10 +5,13 @@
...
     "require": {
-        "php": "^7.4 || ^8.0",
+        "php": "^7.3 || ^7.4 || ^8.0",
         "sylius/sylius": "^1.10",
         "drewm/mailchimp-api": "^v2.5.4",
-        "ext-json": "*"
+        "ext-json": "*",
+        "vimeo/psalm": "^4.12",
+        "composer/xdebug-handler": "^2.0",
+        "bitbag/coding-standard": "^1.0"
```
Comment (oallain) [inline composer.json:14]: Why ???

**Pull request merged**